reference/3.0/Microsoft.PowerShell.Core/About/about_Ref.md

```diff
@@ -5,72 +5,156 @@ locale:  en-us
 keywords:  powershell,cmdlet
 title:  about_Ref
 ---
-# About Ref
-
-## SHORT DESCRIPTION
 
-Describes how to create and use a reference variable type.
+# About Ref
 
-## LONG DESCRIPTION
+## Short description
 
-You can use the reference variable type to permit a method to change the value
+Describes how to create and use a reference type variable. You can use
+reference type variables to permit a method to change the value
```

Comment:
Let's change 'method' to 'function' throughout this article. All of the examples are functions. We are never showing them in the context of a PowerShell class definition. So I think it is confusing to call it a method.

Reply:
check